package/MDAnalysis/converters/RDKit.py

```diff
@@ -53,7 +53,7 @@
     infers the structures of approximately 90% of the `ChEMBL27`_ dataset.
```

Comment:
Should this be updated to 
> 99.1% of the 1.7M compounds in the CHEMBL27 dataset

Reply:
Is the whole warning still necessary or can it become a Note or just text?

Reply:
> Should this be updated to
> 
> > 99.1% of the 1.7M compounds in the CHEMBL27 dataset

Yes! good catch

> Is the whole warning still necessary or can it become a Note or just text?

I think it's necessary to keep the warning, just to catch people's attention that it's not a perfect solution and there are a few cases that are known to fail.